src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
+@keyframes rotates {
+  from {
+    rotate: 0;
```

Comment:
The correct property for rotation is `transform: rotate(...)`, not `rotate`. So, you should change `rotate: 0;` to `transform: rotate(0deg);`.